src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Shooter extends SubsystemBase {
+  /** Creates a new Shooter. */
```

Comment:
I forgot to mention this but each method + the constructor in this class should have javadocs -- use the @param @return @link annotations whenever applicable. You can look at the file LocalADStarAK in this repository for reference, or at the code in the 2023-Robot-Code repository in the lobstahbots organization on GitHub. (Some of the files in this repo don't have javadocs yet and adding them is one of my todos.) Let me know if you're confused about anything!